plugins/torrent-tracker.js

```diff
@@ -0,0 +1,34 @@
+var Server = require('bittorrent-tracker').Server 
+
+function isFunction (f) {
+  return 'function' === typeof f
+}
+
+function isString (s) {
+  return 'string' === typeof s
+}
+
+exports.name = 'torrent-tracker'
+exports.version = '1.0.0'
+
+//trust-proxy 'x-forwarded-for' header from reverse proxy 
+//tell clients to announce on this interval (ms) 
+exports.manifest = {
```

Comment:
well how I did it in other places was take a sub config object off the global config
https://github.com/ssbc/scuttlebot/blob/master/plugins/gossip.js#L56
and then have inline defaults https://github.com/ssbc/scuttlebot/blob/master/plugins/gossip.js#L160

that does make it a bit difficult to see what the settings that plugin would use.
If we have a lot of plugins with config it might be better to have a config section on the plugin.
But for now it might be simplest to take the subobject, and then set defaults on it.